raft/raft.go

```diff
@@ -718,14 +717,10 @@ func stepLeader(r *raft, m pb.Message) {
 				return
 			}
 			r.abortLeaderTransfer()
-			r.logger.Infof("%x [term %d] abort transfer leadership to %x", r.id, r.Term, lastLeadTransferee)
+			r.logger.Infof("%x [term %d] abort previous transferring leadership to %x", r.id, r.Term, lastLeadTransferee)
 		}
 		if leadTransferee == r.id {
-			if lastLeadTransferee == None {
-				r.logger.Debugf("%x is already leader. Ignored transfer leadership to %x", r.id, r.id)
-			} else {
-				r.logger.Debugf("%x abort transfer leadership to %x, transfer to current leader %x.", r.id, lastLeadTransferee, r.id)
-			}
+			r.logger.Debugf("%x is already leader. Ignored transferring leadership to self", r.id)
```

Comment:
Those two `r.logger.Debugf` are just used for printing debug information.
test case `TestLeaderTransferToSelf` is for if case.
test case `TestLeaderTransferBack` is for else case.

We also can combine these two. If we combine them, variable `lastLeadTransferee` can also be removed.